### PR TITLE
Feature/smc fork

### PR DESCRIPTION
**Pull request by ARR552**
### What does this PR do?

Updates the smc with the forkId change

### Reviewers

Main reviewers:

- @cool-develope

### CODE REVIEW

**Review by cool-develope — APPROVED** [review]
When do we merge this update to `develop`?

**Comment by ARR552** [comment]
> When do we merge this update to `develop`?

Ones we create the next release candidate. We will merge release branch to develop

File: test/operations/manager.go [hunk 1]
```diff
@@ -488,19 +488,19 @@ func (m *Manager) CheckAccountTokenBalance(ctx context.Context, network NetworkS
 }
 
 // GetClaimData gets the claim data
-func (m *Manager) GetClaimData(ctx context.Context, networkID, depositCount uint) ([][bridgectrl.KeyLen]byte, *etherman.GlobalExitRoot, error) {
+func (m *Manager) GetClaimData(ctx context.Context, networkID, depositCount uint) ([bridgectrl.KeyLen][bridgectrl.KeyLen]byte, *etherman.GlobalExitRoot, error) {
```
Comment (C001-developer) [inline test/operations/manager.go:491]: we are not able to `KeyLen` as the 1st dimension, it is same as with `MerkleTree.Height` (not 32)

File: test/operations/manager.go [hunk 2]
```diff
@@ -488,19 +488,19 @@ func (m *Manager) CheckAccountTokenBalance(ctx context.Context, network NetworkS
...
 	res, err := m.bridgeService.GetProof(context.Background(), &pb.GetProofRequest{
 		NetId:      uint32(networkID),
 		DepositCnt: uint64(depositCount),
 	})
 	if err != nil {
-		return nil, nil, err
+		return [32][32]byte{}, nil, err
```
Comment (C001-developer) [inline test/operations/manager.go:497]: same, I can see several cases regarding the height, this value is customizable through the config

File: test/operations/manager.go (hunk 1 above)
Comment (ARR552) [reply test/operations/manager.go]: As this height can not be defined as variable, I have decided to define it as constant. If this is not good enough we could create an issue to not forget it.

Comment (C001-developer) [reply test/operations/manager.go]: OK, fair enough, this is only limited to the test toolkit, I think we can ignore

**Review by C001-developer — APPROVED** [review]

**Pull request merged**